src/awesome.coffee

```diff
@@ -1,237 +1,211 @@
+    getDeviceById id, robot, (error, device, devices) ->
+      index = devices.indexOf(device)
+      if error
+        slackCallback error
+      else
+        device.status = 'available'
+        device.user = ''
+        device.date = ''
+
+        devices[index] = device
+        devices = robot.brain.set devicesArrayKey, devices
+        slackCallback null, "It's back!"
+
+  removeDevice = (id, robot, callback) ->
+    getDeviceById id, robot, (error, device, devices) ->
+      if device
+        index = devices.indexOf(device)
+        if error
+          slackCallback error
+        else
+          devices.splice index, 1
+          devices = robot.brain.set devicesArrayKey, devices
+          slackCallback null, "Removed"
+      else
+        slackCallback null, "Error removing"
 
-  executeCommand = (text, user, callback) ->
+  executeCommand = (msg, robot, callback) ->
+    text = msg.message.text
+    user = msg.message.user
+    console.log text
     async.waterfall [
       (cb) ->
         if text == null || text == 'undefined' || text == undefined
           console.log text
           cb 'error'
         else cb null, text.split(' ')
       (args, cb) ->
-        helpText = "Yo " + user.name + "! How r u doin, bro? Im here to help you find and manage devices. For example, an iPhone 6 Plus iOS 9.2 64 GB White. \n\nYou can type \`want\` followed by any term of what you want to find (e.g., \*want iPhone\* or \*want iOS\*). \n\nOr, you can just ask for the availability of all devices by typing \`want all\`. When you take a device, \nremember to tell others by typing \`got\` followed by its id. When you return a device, \ntell your bros too: type \`back\` followed by its id.\n\nYou can also register a new device or delete an existing one by typing \`register\` or \`delete\`, followed by its full information. Remember, bro, You will need: \*model\*, \*os\*, \*version\*, \*notes\*, \*owner\* (e.g., register \"Blackberry Curve\" \"blackberry\" \"7.0.0\" \"Bundle 2055 black\" \"your name\")."
+        args.splice(0, 1);
+        helpText = "Yo " + user.name + "! How r u doin, bro? Im here to help you find and manage devices. For example, an iPhone 6 Plus iOS 9.2 64 GB White. \n\nYou can type \`want-device\` followed by any term of what you want to find (e.g., \*want-device iPhone\* or \*want-device iOS\*). \n\nOr, you can just ask for the availability of all devices by typing \`want-device all\`. When you take a device, \nremember to tell others by typing \`got-device\` followed by its id. When you return a device, \ntell your bros too: type \`back-device\` followed by its id.\n\nYou can also register a new device or delete an existing one by typing \`register-device\` or \`delete-device\`, followed by its full information. Remember, bro, You will need: \*id\*, \*model\*, \*os\*, \*version\*, \*notes\*, \*owner\* (e.g., register-device \"Blackberry 1\" \"Blackberry Curve\" \"blackberry\" \"7.0.0\" \"Bundle 2055 black\" \"your name\")."
```

Comment:
@askmon 👍  por colocar `device-` no início, mas agora o help ficou desatualizado 😢